vendor-extra/LiberoPatternsBundle/src/constants.php

```diff
@@ -8,8 +8,13 @@
 const PAGE_GRID_MAIN = 'main';
 const PAGE_GRID_END = 'end';
 
-const CONTENT_GRID_PRIMARY = 'primary';
+const MAIN_GRID_FULL = 'full';
+const MAIN_GRID_MAIN = 'main';
+const MAIN_GRID_PRIMARY = 'primary';
```

Comment:
Based on the assumption that _all_ 'main' grids have `full`, `main`, and `primary` areas. /cc @davidcmoulton